app/controllers/teams/users_controller.rb

```diff
@@ -1,5 +1,33 @@
 class Teams::UsersController < AuthenticatedController
-   before_action -> { authorize team, policy_class: TeamUsersPolicy }
+    before_action -> { authorize team, policy_class: TeamUsersPolicy }
+
+    def new
+      @user = User.new
+    end
+  
+    def create
+      add_email = user_params.require(:email).downcase
+      if not valid_email_address?(add_email)
```

Comment:
```suggestion
      if not add_email.match?(Devise.email_regexp)
```
mostly because the `match?` method will return true or false (see comment below: https://github.com/18F/identity-dashboard/pull/581/files#r1006113751)